src/utils/constants.ts

```diff
@@ -64,10 +64,12 @@ export const TIER_DATA: { [x: number]: string[] } = {
 };
 export const NAME_PRICE_INFO =
   'Registration fees are determined by the character length of the domain, lease duration, and what tier you choose.';
+export const PERMABUY_INFO =
```

Comment:
the mulitpliers here will be set in state

Reply:
updated to be a function that returns based on provided contract info, manually set for now